input/docs/handbook/when-activated/index.md

```diff
@@ -1,37 +1,36 @@
-# WhenActivated (Object Lifecycle Management)
-
 # ViewModel
 
-Geoff:
-> who uses ISupportsActivation on their view models and why. Is it just to get this.WhenActivated and have a way to track viewmodel disposables? Ie exactly same concern as the view, just a pattern for doing it in VM.
-
-JCM:
-> We use it. Why? To defer the setup of the view model until its truly required. It also allows us 2 stop updating of the view model based e.g. things like an observable updating the users current location.
-
-Michael:
-> We use it for tracking disposables. For example, a background task that periodically pings a network endpoint for some data needs to be disposed of when view is deactivated and re-set-up again when it is activated.(edited)
-
-
-Here's how it is implemented.
+WhenActivated is a way to track disposables. Behind that, it can be used to defer the setup of a ViewModel until it's truly required. It also gives us an ability to stop reacting to hot observables, like a background task that periodically pings a network endpoint or an observable updating users current location. See an example:
```

Comment:
Well WhenActivated() has the ability to track disposables, but also happens when the ViewModel is activated so you can do startup logic as well in here. 

eg somethike

```cs
this.WhenActivated(disposable => 
{
    this.TextBox = "I'm brand new";
    disposable(Disposable.Create(() => this.TextBox = "bye");
});
```